### PR TITLE
Blog post on PrestoSummit Bangalore

### PR DESCRIPTION
**Pull request by vijaymann**

### CODE REVIEW

File: _posts/2019-09-05-Presto-Summit-Bangalore.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+# Keynote by Martin, Dain and David
+[Slides](https://go.qubole.com/rs/510-QPZ-296/images/Presto%20Summit%20India%20-%201.%20Keynote%20by%20Martin%2C%20David%2C%20Dain.pdf)
+[Video](https://youtu.be/viBY8Fa3OjI)
+
+![](/assets/blog/Bangalore-2019/JE1A1911.JPG)
+
+This was followed by the most awaited presentation of the day - 
+the keynote from Martin, Dain and David. Martin took the audience through Presto's journey - right from its birth at Facebook, 
+to its growth and adoption at Facebook, and finally to the present with the formation of PrestoSql Foundation
```
Comment (martint) [inline _posts/2019-09-05-Presto-Summit-Bangalore.md]: `Prestosql Foundation` -> `Presto Software Foundation`

File: _posts/2019-09-05-Presto-Summit-Bangalore.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+Final presentation of the day was from [Rohit Srivastava](https://www.linkedin.com/in/rohitsrivastava20/), 
+Engineering Manager at [MiQ Digital](http://www.wearemiq.com/), who presented an overview of Unified Insights & Data 
+Analytics platform at MiQ. He highlighted several challenges that his team had to overcome, such as scaling the 
+team/infrastructure/company, dealing with data copies, duplication of data pre-processing and the cost and 
+effort that goes into it, meeting strict SLAs etc.  He gave an overview of how using Presto on Qubole for all 
+dashboarding needs with additions like standardising most of their data to be stored in the Apache Parquet format 
+on S3 has helped overcome some of these challenges.
+
+
+In summary, first Presto Summit in India, had a great mix of  talks - some  were around presto usage and
```
Comment (martint) [inline _posts/2019-09-05-Presto-Summit-Bangalore.md]: presto -> Presto

File: _posts/2019-09-05-Presto-Summit-Bangalore.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+Engineering Manager at [MiQ Digital](http://www.wearemiq.com/), who presented an overview of Unified Insights & Data 
+Analytics platform at MiQ. He highlighted several challenges that his team had to overcome, such as scaling the 
+team/infrastructure/company, dealing with data copies, duplication of data pre-processing and the cost and 
+effort that goes into it, meeting strict SLAs etc.  He gave an overview of how using Presto on Qubole for all 
+dashboarding needs with additions like standardising most of their data to be stored in the Apache Parquet format 
+on S3 has helped overcome some of these challenges.
+
+
+In summary, first Presto Summit in India, had a great mix of  talks - some  were around presto usage and 
+experience of operating large presto deployments across multiple clouds, while some others focussed on niche
```
Comment (martint) [inline _posts/2019-09-05-Presto-Summit-Bangalore.md]: presto -> Presto

File: _posts/2019-09-05-Presto-Summit-Bangalore.md (hunk 1 above)
Comment (vijaymann) [reply _posts/2019-09-05-Presto-Summit-Bangalore.md]: done

File: _posts/2019-09-05-Presto-Summit-Bangalore.md (hunk 2 above)
Comment (vijaymann) [reply _posts/2019-09-05-Presto-Summit-Bangalore.md]: done

File: _posts/2019-09-05-Presto-Summit-Bangalore.md (hunk 3 above)
Comment (vijaymann) [reply _posts/2019-09-05-Presto-Summit-Bangalore.md]: done

**Comment by vijaymann** [comment]
Took care of the above corrections and also reduced the resolution of images

**Pull request merged**